pm2-windows.json

```diff
@@ -1,14 +1,25 @@
 {
   "apps": [
+   {
+      "name": "@ove/ove-asset-manager",
+      "cwd": "./packages/ove-asset-manager",
+      "kill_timeout": 3000,
+      "restart_delay": 3000,
+      "script": "dotnet",
+      "args": "dist/OVE.Service.ImageTiles.dll --server.urls=http://0.0.0.0:8181",
```

Comment:
Why does the asset manager refer to the ImageTiles dll? I don't quite understand how this actually manages to work. I think you are safe within the docker container since it only uses the pm2.json but not this one.